Packs/DigitalGuardian/Integrations/DigitalGuardianARCEventCollector/DigitalGuardianARCEventCollector.py

```diff
@@ -13,6 +13,8 @@
 VENDOR = 'Digital Guardian'
 PRODUCT = 'ARC'
 
+EventsGenerator = Generator[dict[str, Any], None, None]
```

Comment:
nice touch, I'm a bit worried the typedef here may create more confusion (dev will have to figure out what's a EventsGenerator, also the events aren't an existing type here) than just using the `Generator[dict[str, Any], None, None]` directly in code.

Your call :)

Reply:
Updated to use lists as we discussed in commit ef814676d12876d5adc3cde92e5f7ff92c595e24